docs/ref/templates/builtins.txt

```diff
@@ -711,6 +711,12 @@ available to the included template::
     been evaluated and rendered* - not blocks that can be overridden by, for
     example, an extending template.
 
+.. note::
+    If the included template is missing, the effect varies depending on the
+    :setting:`DEBUG` setting: when ``DEBUG`` mode is on, a
```

Comment:
In my testing it was toggled on / off by DEBUG setting, perhaps it depends on both? I will test later today.